src/Lesson2.java

```diff
@@ -173,10 +171,9 @@ public static int isMinElementArray(int[] array) {
         return min;
     }
 
-    public static void isTwoDimensionalRandomArray(int n, int m) {
+    public static void createTwoDimensionalRandomArray(int n, int m) {
         int max = 100;
         int min = -100;
-        max -= min;
         int[][] arrays = new int[n][m];
         for (int i = 0; i < n; i++) {
             for (int j = 0; j < m; j++) {
```

Comment:
there's no need to print the array
and it's better to use Random than Math.random